packages/pegasus/globals.d.ts

```diff
@@ -1,6 +1,7 @@
 interface VatData {
```

Comment:
Are we really repeating this everywhere? Please no.

Reply:
Just keeping up with the status quo.  It's deeply, deeply irritating.  I will file a ticket on this.

Reply:
?

Reply:
Ticket filed.

Reply:
#4830 for anyone watching